server/src/main/java/com/unosquare/carmigo/model/request/CreateSearchJourneysCriteria.java

```diff
@@ -0,0 +1,27 @@
+package com.unosquare.carmigo.model.request;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+import javax.validation.constraints.NotNull;
+import java.time.Instant;
+
+@Data
+public class CreateSearchJourneysCriteria
+{
+    @NotNull
+    @JsonProperty("locationIdFrom")
+    final int locationIdFrom;
```

Comment:
hey man.  int can never be null.  You can choose to use the wrapper (Int) or @Range(min = 1)

Reply:
I added @NotNull because I understand params are `String` which, in this case, are parsed into `int`.
I wasn't aware of @Range. I'm replacing with this.